apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9582,6 +9584,11 @@ span#HW_badge_cont:nth-child(3) {
 .manage-group-account-form .button-group .btn{
     margin-right: 15px;
 }
+.inventory-page{
+    .daterangepicker{
```

Comment:
Ignore class naming convention because daterangepicker also added on TS file and having a 875 result in 116 files